.travis.yml

```diff
@@ -2,7 +2,9 @@ language: python
 
 python: 
 - "3.6.6"
-
+matrix:
```

Comment:
Please share the reason for adding this to the .travis file.

Reply:
Oh I was trying to add Flake 8 following some tutorial, let me remove it please